### ISSUE TITLE
Sailsjs Validate one model atribute only.

### ISSUE THREAD

**Issue by netrevisanto** [balderdashy/sails#2324]
Hello everyone.
I am pleased to use this framework for a lot of projects and it's simply fantanstic.
Today i had a situation where i was in need to validate just some attributes of my model. For example, in my user model, i want to validate just the user password and confirmation attributes, instead of causing a global validation, by calling Model.update. Maybe there is a Model.validate or Model.validadeAttribute method that i am missing?

My question is,  how can this be done, without developing new code, by using the built in methods?
And thanks for this wonderful tool that you guys are mantaining.

**Comment by CWyrtzen** [comment]
@netrevisanto has your question been resolved?

**Comment by sailsbot** [comment]
Thanks for posting, @netrevisanto.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
